### DISCUSSION TITLE
Display Port Not Working in 0.7.0

### DISCUSSION THREAD

**Discussion by soeryobadja** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#121] [Q&A]
After using 0.7.0 version in Catalina 10.15.7, I just found out that the DP port is not working well. When DP Port connected, it projecting display, but the main monitor then goes dark. Disconnecting the DP external monitor does not make the laptop monitor showing back the display. While the HDMI display work perfectly. 
Anybody has similar issue? 

Thanks.

**Comment by duszmox** [comment]
Hi, can you try out this config.plist and tell me what you experience?
[Config.plist.zip](https://github.com/simprecicchiani/ThinkPad-T460s-macOS-OpenCore/files/7172439/Config.plist.zip)
It's an 0.7.3 config file, it won't work on older versions
